src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -31,7 +31,8 @@ var EventEntries = React.createClass({
     exception: require("./interfaces/exception"),
     request: require("./interfaces/request"),
     stacktrace: require("./interfaces/stacktrace"),
-    template: require("./interfaces/template")
+    template: require("./interfaces/template"),
+    csp: require("./interfaces/csp"),
```

Comment:
It's not invalid. Only IE7 and below throw error on this.

On Monday, October 12, 2015, David Cramer <notifications@github.com> wrote:

> In src/sentry/static/sentry/app/components/events/eventEntries.jsx
> <https://github.com/getsentry/sentry/pull/2154#discussion_r41730286>:
>
> > @@ -31,7 +31,8 @@ var EventEntries = React.createClass({
> >      exception: require("./interfaces/exception"),
> >      request: require("./interfaces/request"),
> >      stacktrace: require("./interfaces/stacktrace"),
> > -    template: require("./interfaces/template")
> > +    template: require("./interfaces/template"),
> > +    csp: require("./interfaces/csp"),
>
> trailing comma is invalid // @benvinegar <https://github.com/benvinegar>
> how do we get lint for this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getsentry/sentry/pull/2154/files#r41730286>.
>